modules/ROOT/pages/connecting-salesforce-orgs.adoc

```diff
@@ -0,0 +1,79 @@
+
+In some cases, the Anypoint Platform organization administrator and Salesforce organization administrator are different people, and in some cases, they are the same person.
+
+== Before You Begin
+
+Anypoint Platform organization administrators must have:
+
+* Anypoint Platform Organization Administrator permission at the root organization level
+* Salesforce tenant key of the Salesforce organization to connect
++
+The Anypoint Platform organization administrator can get the tenant key from the Salesforce organization administrator.
+
+Salesforce organization administrators must have:
+
+* Salesforce administrator permission set
+* Anypoint Platform organization key
++
+The Salesforce organization administrator can get the Anypoint Platform organization key from the Anypoint Platform organization administrator. 
+
+[connect-orgs]
+== Connect With a Salesforce Org in Anypoint Platform
+
+As an Anypoint Platform organization administrator, you can start the process of connecting to a trusted Salesforce organization. 
+
+You must have the Salesforce organization's connection key (tenant key) before you can connect to a Salesforce organization. Contact the Salesforce organization admin to get the tenant key. 
+
+include::partial$include-nav-salesforce.adoc[]
+
+. Click *Add Salesforce Org*.
+. Enter the Salesforce organization's tenant key, and click *Confirm*. 
++
+The format for the Salesforce tenant key is `cloud/env/org-id`.
+. In *Add the Salesforce Org*, verify the information and click *Add*. 
+. Click *Copy* to copy the Anypoint Platform organization key. 
+. If you are the admin for the Salesforce organization you are adding, click *Manage in Salesforce* to continue setting up the relationship in Salesforce. 
++
+If you are not the Salesforce admin, provide the Anypoint Platform organization key to the Salesforce admin so they can finish the connection process in Salesforce.
+
+When the connection process is complete, the Salesforce organization appears in the *Salesforce Orgs* section of the *Salesforce* page.
+
+[complete-organization-connection]
+== Complete the Organization Connection in Salesforce
+
+As a Salesforce organization administrator, you complete the connection between your organization and the Anypoint Platform organization when the connection process is initiated in Anypoint Platform. You must have the Anypoint Platform organization key to complete the connection.
+
+include::partial$include-salesforce-mulesoft-setup-nav.adoc[]
+
+. In *Your connected Anypoint Platform Organization*, click *View Connection Info*.
+. In *Connection to MuleSoft Anypoint Platform*, enter the Anypoint Platform organization key in *Anypoint Platform Organization ID*, and then click *Connect*.
++
+NOTE: The Anypoint Platform organization key is the universally unique identifier (UUID) for the Anypoint Platform organization ID.
```

Comment:
The organization key is not the organization UUID. It is also in the format `cloud/env/org-id`